src/main/java/com/wiley/autotest/spring/SeleniumTestExecutionListener.java

```diff
@@ -252,13 +252,15 @@ public void prepareTestInstance(final TestContext context) throws Exception {
                 SeleniumHolder.setIOSDriver(iosDriver);
             }
 
+            //TODO VE, NT - consider reworking with new model in mind (Visible/Dom/Null)
+
             String classFromSettings = settings.getProperty("our.webelement.class");
             if (classFromSettings != null && !classFromSettings.isEmpty()) {
                 SeleniumHolder.setOurWebElementClass(classFromSettings);
             } else if (configuration.getClassOfElement() != null) {
                 SeleniumHolder.setOurWebElementClass(configuration.getClassOfElement().getName());
             } else {
-                SeleniumHolder.setOurWebElementClass("com.wiley.autotest.selenium.elements.upgrade.TeasyElement");
+                SeleniumHolder.setOurWebElementClass("com.wiley.autotest.selenium.elements.upgrade.VisibleTeasyElement");
```

Comment:
@estuold let's discuss on how we should implement this with the new elements (Visible, Dom, Null)